ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -98,7 +98,7 @@
                 <span>You've submitted too many wrong responses. Please <a href="@mailtoUrl">use this link to email us your answer for @Model.Puzzle.PlaintextName or ask for help</a>.</span>
             </div>
         }
-        else if (Model.PuzzleState.IsTeamLockedOut)
+else if (Model.PuzzleState.IsLockedOut)
```

Comment:
Nit: It looks like something happened to indentation here

Reply:
fixed in newest iteration